velox/docs/functions/spark/math.rst

```diff
@@ -233,3 +233,16 @@ Mathematical Functions
 .. spark:function:: unaryminus(x) -> [same as x]
 
     Returns the negative of `x`.  Corresponds to Spark's operator ``-``.
+
+.. spark:function:: unhex(x) -> varbinary
+
+    Converts hexadecimal varchar ``x`` to varbinary.
+    If size ``x`` is odd, parses the first character and puts it at index 0 of binary result,
```

Comment:
This description is a bit difficult to understand and it doesn't mention that sometimes the results has an extra zero byte at the end. Perhaps,

```
'x' is considered case insensitive and expected to contain only hexadecimal characters 0-9 and A-F. If 'x' contains non-hexadecimal character, the function returns NULL.

When 'x' contains even number of characters, each pair is converted to a single byte. The number of bytes in the result is half the number of bytes in the input.

When 'x' contains a single character, the result contains a single byte whose value matches the hexadecimal character.

When 'x' contains an odd number characters greater than 2, the first character is ignored, the remaining pairs of characters are converted to bytes, then zero byte is added at the end of the output.
```

It would be helpful to also add an example where 'x' contains only 1 bytes and that byte is a valid hexadecimal character.

Reply:
thanks for the better doc explanation!

> It would be helpful to also add an example where 'x' contains only 1 bytes and that byte is a valid hexadecimal character.

I add this example in doc: 
    SELECT unhex("f"); -- \x0F